stacker_blueprints/empire/controller.py

```diff
@@ -0,0 +1,177 @@
+from troposphere import (
```

Comment:
yea i did it in separate commits but i also changed things within the file. /shrug